### DISCUSSION TITLE
manifest.json is not being output

### DISCUSSION THREAD

**Discussion by JoshClose** [statiqdev/Discussions#44] [Q&A] [answered]
I have some files in the root of `/input` like `.nojekyll`, `browserconfig.xml`, and `manifest.json`. The other files are being copied to the output, but `manifest.json` is not. Is this because it's a json file? How do I get it to copy?

**Comment by daveaglick** [comment, marked as answer]
The underlying reason is that JSON files in Statiq Web are treated as data files and processed by the [data pipeline](https://statiq.dev/web/content-and-data/data) by default. [This page](https://statiq.dev/web/content-and-data/) has more information on which files are processed by which pipeline and how to adjust the behavior on a per-file or whole folder basis.

Specifically, the easiest way to treat a single JSON file as an asset and copy it instead of process it is to add a [sidecar file](https://statiq.dev/web/content-and-data/metadata/sidecar-files) (which acts like [front matter](https://statiq.dev/web/content-and-data/metadata/front-matter) but can be specified outside the target file) that sets `ContentType` to `Asset` for the JSON file.

In your case: `_manifest.yml`:
```yml
ContentType: Asset
```

**Reply by JoshClose** [reply]
Thanks. I just came across Statiq and it looks amazing. I'm converting my documentation site over to it.
